pkg/plugin/bundle_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("Bundle", func() {
 				{p1, p2, p3},
 				{p1, p3, p4},
 			} {
-				b, err := NewBundle(name, version, "", plugins...)
```

Comment:
Can we live it as it is and add a new test for a new function using the Options?